config/action.d/firewallcmd-common.conf

```diff
@@ -5,72 +5,102 @@
 
 [Init]
 
+# Option:  bantime
+# Notes:   Specifies the bantime in seconds (handled internally rather than by fail2ban)
+# Values:  [ NUM ]  Default: 600
+
+bantime = 600
```

Comment:
Seeing as though an incrementing bantime doesn't matter in firewallcmd-ipset unban or any other firewallcmd at the time being, does anyone see a problem in it being defined in -common?